<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Port Sender</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="m_btn_connect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>152</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть порт</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="m_grp_port">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Порт</string>
    </property>
    <widget class="QComboBox" name="m_cmbox_port_speed">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>153</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="m_cmbox_port">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>19</y>
       <width>152</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="m_grp_process">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>351</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="m_radio_role_sender">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Передатчик</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="m_radio_role_reciver">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Приемник</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="Line" name="m_line_marker">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>351</width>
       <height>8</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QGroupBox" name="m_grp_send">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>341</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="m_lbl_ms">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>мс</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_edit_send_ms">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>200</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m_btn_start_send">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Начать передачу с интервалом</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="m_grp_stat_send">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>321</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Статистика</string>
      </property>
      <widget class="QLabel" name="m_lbl_send_lbl">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Передано пакетов</string>
       </property>
      </widget>
      <widget class="QLabel" name="m_lbl_send">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>30</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="m_grp_rcv">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>341</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="m_grp_stat_rcv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>321</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Статистика</string>
     </property>
     <widget class="QLabel" name="m_lbl_rcv_lbl">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Принято пакетов</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_lbl_rcv">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_lbl_err_lbl">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ошибочных байтов</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_lbl_err">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_lbl_shift">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_lbl_shift_lbl">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Сдвиг данных (бит)</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="m_btn_reset_stat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Сброс статистики</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btn_dbg">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>DBG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btn_dbg_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>DBG2</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
